src/AirPyllution/AirPyllution.py

```diff
@@ -0,0 +1,45 @@
+
+def get_pollution_history(start_date, end_date, lat, lon, api_key):
+    """Returns a dataframe of pollution history for a location between a specified date range
+    
+    Given a specified date range, the function makes an API request to the OpenWeather Air Pollution API and fetches
+    historic pollution data for a given location.
+
+    The function transforms the returned JSON object from the request into a Pandas dataframe.
+
+    Note: Historical data is accessible from 27th November 2020
+
+    Parameters
+    ----------
+    start_date : int
+        start date of the time frame as a UNIX timestamp, e.g. 1606488670
+    end_date : int
+        end date of the time frame as a UNIX timetamp, e.g. 1606747870
+    lat : float
+        geographical latitude coordinate for the location
+    lon : float
+        geographical longitude coordinate for the location
+    api_key: string
+        OpenWeather API key
+    Returns
+    -------
+    pandas.DataFrame
+        a dataframe of the data returned from Air Pollution API - columns are as followed:
+        ==========  ==============================================================
+        date        int
```

Comment:
This might not be necessary for week 1, but I was envisioning using the datetime package to transform the dates into something human readable

Reply:
Thanks! I made an issue https://github.com/UBC-MDS/airpyllution/issues/12 for us to do this post week 1